vtr_flow/tasks/regression_tests/vtr_reg_strong/strong_read_rr_graph/config/config.txt

```diff
@@ -0,0 +1,27 @@
+#
```

Comment:
The "verify_rr_graph" tests do what you want: https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/aec608ff60a0c01b3c93407b722684d810f04b23/vtr_flow/tasks/regression_tests/vtr_reg_strong/strong_verify_rr_graph_bin/config/config.txt#L29

The `-verify_rr_graph` flag writes out a rr graph and reads it back and and verifies its the same rr graph.

Reply:
https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/aec608ff60a0c01b3c93407b722684d810f04b23/vtr_flow/scripts/run_vtr_flow.py#L576-L578
https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/08f054c85e22ddf33811d91b2dd45daf5ee2341e/vtr_flow/scripts/python_libs/vtr/vpr/vpr.py#L271-L299

Use this structure also means you don't need to add the rr graph to the repo.

Reply:
Great. I have changed the test to use `-verify_rr_graph` feature. Now we just need a test `verify_rr_graph_titan` to do the job.